telethon/client/updates.py

```diff
@@ -415,18 +415,60 @@ async def _update_loop(self: 'TelegramClient'):
                 except GapError:
                     continue  # get(_channel)_difference will start returning requests
 
-                updates_to_dispatch.extend(self._preprocess_updates(processed, users, chats))
+                updates_to_dispatch.extend(await self._preprocess_updates(processed, users, chats))
         except asyncio.CancelledError:
             pass
         except Exception as e:
             self._log[__name__].exception('Fatal error handling updates (this is a bug in Telethon, please report it)')
             self._updates_error = e
             await self.disconnect()
 
-    def _preprocess_updates(self, updates, users, chats):
-        self._mb_entity_cache.extend(users, chats)
-        entities = {utils.get_peer_id(x): x
-                    for x in itertools.chain(users, chats)}
+    async def _fetch_peers(self, users: list, chats: list):
+        result_users = {}
+        result_chats = {}
+        is_min = False
+        for peer in itertools.chain(users, chats):
+            if getattr(peer, 'min', False):
+                peer_id = peer.id
+                if not self._mb_entity_cache.get(peer_id) and peer.id != 136817688:
```

Comment:
The final version should definitely not contain hardcoded user IDs.

Reply:
This id is a Channel_Bot that, as far as I saw, is not hashed and not contained in channel difference, because of this, every time it comes across, it forces GetChannelDifference to run